clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/LeaderFollowerPartitionStateModel.java

```diff
@@ -127,7 +129,15 @@ public void onBecomeOfflineFromStandby(Message message, NotificationContext cont
 
   @Transition(to = HelixState.DROPPED_STATE, from = HelixState.OFFLINE_STATE)
   public void onBecomeDroppedFromOffline(Message message, NotificationContext context) {
-    executeStateTransition(message, context, this::removePartitionFromStoreGracefully);
+    executeStateTransition(message, context, () -> {
+      try {
+        // Gracefully drop partition to drain the requests to this partition
+        Thread.sleep(TimeUnit.SECONDS.toMillis(getStoreConfig().getPartitionGracefulDropDelaySeconds()));
```

Comment:
Might be clearer to move this inside removePartitionFromStoreGacefully and explain that this is to give time for the router to stop sending requests

Reply:
I think we should remove or change the waiting time in this function:
https://github.com/linkedin/venice/blob/main/clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java#L873

The wait in this function is mainly to try to drain the ingestion task, which shouldn't need 10mins.

Putting the wait here is fine with enough comments and the function in `KafkaStoreIngestionService` shouldn't leverage the same config for waiting and it should be much shorter.

Reply:
@gaojieliu That function doesn't wait for the entire 10 minutes actually; it's a loop with 10 iteration with 1 minute sleep time only if the draining is not completed; and we break the loop early if draining is completed. That function looks okay to me.

Reply:
If that is the case, I don't think we should use the same config: 
```
getStoreConfig().getPartitionGracefulDropDelaySeconds()
```
in function: `removePartitionFromStoreGracefully`.
Either we use some hard-coded number in `removePartitionFromStoreGracefully` or use a different config.

Reply:
Updated in the latest diff; I reused an existing config called "server.stop.consumption.wait.retries.num"; Da-Vinci client does use this config "server.stop.consumption.wait.retries.num" for "removePartitionFromStoreGracefully"; and backend doesn't use it currently.